.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "swiftlint.path": "../Pods/SwiftLint/swiftlint"
```

Comment:
Is this setting used by SwftLint plugin? If so, should we also check in the VS Code setting (presumably the `.vscode/extensions.json` file?) where it says the SwiftLint plugin is required for this project?

Reply:
Oh, good point! Thanks. I'm still a VS Code noob.